scripts/pre_push_hook.py

```diff
@@ -352,7 +378,14 @@ def main():
             if files_to_lint:
                 lint_status = _start_linter(files_to_lint)
                 if lint_status != 0:
-                    print 'Push failed, please correct the linting issues above'
+                    print ('Push failed, please correct the linting issues '
```

Comment:
Do we need to break this into two lines? Earlier it was fitting in a single line.
Oh, is it because of the period in the end?

Reply:
It was because of the period :)